contrib/apparmor/template.go

```diff
@@ -38,7 +38,7 @@ profile /usr/bin/docker (attach_disconnected, complain) {
   /dev/urandom r,
   /dev/null rw,
   /dev/pts/[0-9]* rw,
-  /run/balena.sock rw,
+  /run/balena-engine.sock rw,
```

Comment:
By the way, even before any renaming, elsewhere the path was `/var/run/balena.sock` instead of `/run/balena.sock`. Shouldn't it be `/var/run` here as well?